src/doc/trpl/traits.md

```diff
@@ -338,7 +338,8 @@ plain type parameter (like `T`).
 
 ## Default methods
 
-There’s one last feature of traits we should cover: default methods. It’s
+There’s one important feature of traits distinguishing them from "interfaces",
+familiar to many from languages like Java, C# or Go: *default methods*. It’s
```

Comment:
We _generally_ try to leave specific language comparisons out of docs unless it's _really_ important. and we've moved away from using * as highlighting too.